client/src/components/views/AdminPage/UsersOffersPage/AdminUsersOffersPage.styled.tsx

```diff
@@ -0,0 +1,114 @@
+import styled from "styled-components";
+import { FlexWrapper } from "../../../wrappers/FlexCenter/FlexWrapper.style";
+
+export const TableWrapper = styled(FlexWrapper)`
+  overflow-x: scroll;
+  min-height: 80vh;
+  width: 100%;
+  align-items: center;
+  padding: 2rem 1rem;
+
+  .align-right {
+    text-align: right;
+    line-height: 100%;
+  }
+
+  .align-center {
+    text-align: center;
+  }
+
+  .offer-photo {
+    text-align: center;
+  }
+  .offer-photo img {
+    width: 100px;
+    height: 100px;
+  }
+`;
+
+export const Table = styled.table`
+  display: block;
+  max-width: fit-content;
+  margin: 0 auto;
+  overflow-x: auto;
+  white-space: nowrap;
+  background-color: ${({ theme }) => theme.colors.table.primary};
+  border-collapse: collapse;
+
+  tbody tr:nth-child(odd) {
+    background: ${({ theme }) => theme.colors.table.oddRow};
+  }
+
+  tbody tr:hover {
+    background-color: ${({ theme }) => theme.colors.table.secondary};
+  }
+`;
+
+export const THead = styled.thead`
+  font-size: ${({ theme }) => theme.size.desktop.mds}px;
+
+  &:first-child {
+    background-color: #c8c8c8;
+  }
+`;
+
+export const TBody = styled.tbody`
+  font-size: ${({ theme }) => theme.size.desktop.sm}px;
+`;
+
+export const TR = styled.tr``;
+
+export const TH = styled.th`
```

Comment:
naming your variables is important. maybe worth calling differently than TH, TD. tableHeader is cool and generic. If one such component is only in administrator view then maybe administratorTableHeader?